Twilio/Rest/Api/V2010/Account/MessageOptions.php

```diff
@@ -39,14 +39,6 @@ public static function create($from = Values::NONE, $messagingServiceSid = Value
     public static function read($to = Values::NONE, $from = Values::NONE, $dateSentBefore = Values::NONE, $dateSent = Values::NONE, $dateSentAfter = Values::NONE) {
         return new ReadMessageOptions($to, $from, $dateSentBefore, $dateSent, $dateSentAfter);
     }
-
-    /**
-     * @param string $body The body
-     * @return UpdateMessageOptions Options builder
-     */
-    public static function update($body = Values::NONE) {
```

Comment:
Because updating a message body was moved from a optional parameter to a required one i think.